deps/build.jl

```diff
@@ -4,25 +4,25 @@ using BinDeps
 
 blasvendor = Base.BLAS.vendor()
 
+libs = Dict(:direct=>"libscsdir", :indirect=>"libscsindir")
```

Comment:
These names aren't going to change frequently, so just remove this extra level of indirection.